acceptance_tests.sh

```diff
@@ -7,7 +7,7 @@ if [ ! -d $DIR/cob_spec ]; then
     mvn package
 fi
 cd $DIR/SwiftServer
-swift package clean
+#swift package clean
```

Comment:
Just a mistake. Was taking too long, to build cleanly while developing. Uncommented in e8d92e4